cmd/endpoint-ellipses.go

```diff
@@ -288,19 +288,28 @@ func createServerEndpoints(serverAddr string, args ...string) (EndpointZones, in
 		return nil, -1, -1, err
 	}
 
+	// Verify the args setup-type appropriately.
+	{
+		setArgs, err := GetAllSets(args...)
```

Comment:
@harshavardhana this is somehow a duplicate code of the previous block, which I guess can be removed

Reply:
No it is necessary for to avoid calculating different setup types expanding XL to dist XL.

Reply:
> No it is necessary for to avoid calculating different setup types expanding XL to dist XL.

I mean I think you can remove L287, L288 and L289 since you are doing the same thing in L293

Reply:
No it is not the same thing @vadmeste single args to multiple args, we need to pass multiple args to know the setup type.  But only use endpoints for zones later.  That is why both steps are different.